velox/docs/functions/presto/decimal.rst

```diff
@@ -103,8 +103,8 @@ exceeds 38 and cap p at 38 when s <= 38. In this case some operations are reject
 outright while others are allowed to proceed, but may produce runtime errors. Velox
 implementation matches Presto.
 
-Division
---------
+Division and Modulus
```

Comment:
I feels like combining division and modulus may not be ideal. These operations are quite different. For example, division result is approximate, while modulus result is exact. Would you consider adding a separate section for modulus?

Reply:
Moved to separate section, thanks.